tests/TestHelpers/GraphHelper.php

```diff
@@ -554,6 +554,32 @@ public static function getGroup(
 		);
 	}
 
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $adminUser
+	 * @param string $adminPassword
```

Comment:
```suggestion
	 * @param string $user
	 * @param string $password
```
Do we need an admin user to search?